docs/code_ref/offlimb.rst

```diff
@@ -1,7 +1,7 @@
 ***************
-offlimb_enhance
+offlimb
 ***************
```

Comment:
These should match the length of the title.

Reply:
Okay